src/main/java/org/broadinstitute/barclay/help/HelpDoclet.java

```diff
@@ -143,17 +147,16 @@ else if (!settingsDir.isDirectory())
      */
     public static int optionLength(final String option) {
```

Comment:
I don't see this method used anywhere, here or in the gatk pull request.  Is it something we still need?  Is it part of the doclet contract?